spec.md

```diff
@@ -2563,46 +2563,89 @@ Compiler _implementation_.  A FIRRTL Compiler is a program that converts FIRRTL
 text to another representation, e.g., Verilog, VHDL, a programming language, or
 a binary program.
 
-## Aggregate Type Lowering (Lower Types)
+## Module Conventions
 
-A FIRRTL Compiler should provide a "Lower Types" pass that converts aggregate
-types to ground types.
+A module's convention describes how its ports are lowered to the output format,
+and serves as a kind of ABI for modules.
 
-A FIRRTL Compiler must apply such a pass to the ports of all "public" modules in
-a Verilog/VHDL representation.  Public modules are defined as (1) the top-level
-module and (2) any external modules.
+### The "Scalarized" Convention
 
-A FIRRTL Compiler may apply such a pass to other types in a FIRRTL circuit.
+The scalarized convention lowers aggregate ports to ground values. The
+scalarized convention should be the default convention for "public" modules,
+such as the top module of a circuit, "device under test", or an extmodule.
 
-The Lower Types algorithm operates as follows:
+The lowering algorithm for the scalarized convention operates as follows:
 
-1. Ground type names are unmodified.
+1. Ports are scalarized top-to-bottom.
```

Comment:
what does "top to bottom" mean?

Reply:
I've changed this to "in the order they are declared"